packages/react-composites/tests/browser/config.ts

```diff
@@ -22,3 +22,9 @@ export const PARTICIPANT_NAMES = [
   'Adrian Leffler',
   'Schuyler Yost'
 ];
+
+export const IDS = {
```

Comment:
+1

This is the setup I was expecting. Tests define and use the identifiers they want.

<3